x/feegrant/keeper/keeper.go

```diff
@@ -17,14 +18,16 @@ type Keeper struct {
 	cdc        codec.BinaryMarshaler
 	storeKey   sdk.StoreKey
 	authKeeper types.AccountKeeper
+	router     *baseapp.MsgServiceRouter
 }
 
 // NewKeeper creates a fee grant Keeper
-func NewKeeper(cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, ak types.AccountKeeper) Keeper {
+func NewKeeper(cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, ak types.AccountKeeper, router *baseapp.MsgServiceRouter) Keeper {
 	return Keeper{
 		cdc:        cdc,
 		storeKey:   storeKey,
 		authKeeper: ak,
+		router:     router,
```

Comment:
```suggestion
		msgSvcRouter:     router,
```

I would be more specific here, it only needs the service msg router